### ISSUE TITLE
Logframe web view: styling

### ISSUE THREAD

**Issue by jennyhavah** [mercycorps/TolaActivity#1715]
The logframe web view could use some styling attention. Here's the Sketch mockup, for reference.

![image](https://user-images.githubusercontent.com/33670923/59896996-8dd2db00-939f-11e9-9c88-14f6013c2e27.png)

# Styling stuff

## Header area

![image](https://user-images.githubusercontent.com/33670923/59897376-3df51380-93a1-11e9-847b-3a17e54f89a8.png)

- [x] 1. Make page title style consistent with Results framework. Linked program name first, then "Logframe".  Here's the RF page title, for comparison:

     ![image](https://user-images.githubusercontent.com/33670923/59897439-890f2680-93a1-11e9-96e4-ddf8ff75e281.png)

- [x] 2. Move "View results framework" link under page title.

- [ ] ~"Group indicators by" menu: Drop down menu looks a bit scrunched. Can we style it more like the program page?~ _See Paul's comment on this ticket._

     ![image](https://user-images.githubusercontent.com/33670923/59897674-9bd62b00-93a2-11e9-87c3-203ef8c2aa20.png)

## Table

- [x] 1. In terms of row/column background colors and font styling, can we make it similar to the web version of the Indicator plan? Here's a screenshot for reference:

![image](https://user-images.githubusercontent.com/33670923/60306747-4c3db500-98f6-11e9-8654-fa87b68361f6.png)

- [x] 2. Right now, when you widen and narrow the browser window, the column headers get out of alignment with the columns underneath:

![image](https://user-images.githubusercontent.com/33670923/59896602-0c2e7d80-939e-11e9-8f9c-54b1899d73a1.png)

**Comment by axoplasm** [comment]
> "Group indicators by" menu: Drop down menu looks a bit scrunched. Can we style it more like the program page?

Weird but true, this is just a standard html `<select>` element! We use those so rarely it looks a little out of place. I suspect @camgnostic might turn this into a react-select. If not, I'll make a global change to visually align our many, many kinds of select elements

For the table, I changed the markup to use a standard HTML table, (mostly) standard Bootstrap styles. These are the styles we use elsewhere, notably the changelog. **Columns widths are determined entirely by the browser.** This means that for more logframes, the longest columns (indicator names) will try to consume as much real estate as possible. We will need to think carefully about whether we want to override this behavior just for aesthetic purposes. 

I added a little extra line emphasis to separate indicators grouped under a tier (see “Outcome 1: Outcome! in this screenshot:

<img width="1219" alt="image" src="https://user-images.githubusercontent.com/169060/60735991-4e150300-9f0a-11e9-9bc7-727a35220022.png">
